onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -77,7 +78,8 @@ static Env& platform_env = Env::Default();
 
 CustomOpLibrary::CustomOpLibrary(const char* library_path, OrtSessionOptions& ort_so) {
   {
-    OrtPybindThrowIfError(platform_env.LoadDynamicLibrary(library_path, false, &library_handle_));
+    const auto path_str = ToPathString(library_path);
+    OrtPybindThrowIfError(platform_env.LoadDynamicLibrary(path_str, false, &library_handle_));
```

Comment:
`library_path` is copied to the member variable `CustomOpLibrary::library_path_` at the bottom of this constructor. Should the member `library_path_` be made a `PathString` as well? `library_path_` is used for logging purposes if unloading the library handle fails.

Reply:
I will leave it as is so the change does not creep wide. If the class as a whole changes and we can retrofit it to accept `const ORTCHAR_T*` or PathString, then the change should be a part of it.